CMakeLists.txt

```diff
@@ -288,6 +300,14 @@ if(NOT ANDROID AND NOT EMSCRIPTEN AND NOT VITA AND NOT 3DS)
 endif()
 
 
+if(NOT 3DS AND NOT VITA)
+    list(APPEND LIB_SRC
+        ${FILEMAPPER_SRCS}
+        ${STACK_WALKER_SRCS}
```

Comment:
The fact that "stack walker sources" is always empty on non-Windows platforms :smirk_cat: